server/command.go

```diff
@@ -22,10 +22,10 @@ func getCommand() *model.Command {
 	}
 }
 
-func getCommandResponse(responseType, text string) *model.CommandResponse {
+func ephf(format string, args ...interface{}) *model.CommandResponse {
```

Comment:
Huh?

Reply:
Would you rather `e` or `getEphemeralCommandResponsef`? Jokes aside, would do you think `responsef` be intention-revealing enough? I like the short name here for typography, it's only going to be used in this file.

Reply:
Ya `responsef` is good enough.